WordPress/src/main/java/org/wordpress/android/ui/main/MySiteFragment.kt

```diff
@@ -1369,11 +1377,21 @@ class MySiteFragment : Fragment(),
             return
         }
         showQuickStartFocusPoint()
-        val shortQuickStartMessage = stylizeQuickStartPrompt(
-                requireActivity(),
-                activeTutorialPrompt!!.shortMessagePrompt,
-                activeTutorialPrompt!!.iconId
-        )
+        val shortQuickStartMessage =
```

Comment:
NIT: I'm wondering if it might be a little cleaner to build the message separately to remove some of the redundant code, something like: 
```
val actionName = if (activeTutorialPrompt?.task == UPDATE_SITE_TITLE) {
            my_site_title_label.text.toString()
        } else null
        val shortQuickStartMessage =
                    stylizeQuickStartPrompt(
                            requireActivity(),
                            activeTutorialPrompt!!.shortMessagePrompt,
                            activeTutorialPrompt!!.iconId,
                            actionName
                    )
```
However, it's not a big deal so feel free to use what you feel is better :-)

Reply:
Great idea! I implemented it in a bit different way - since `UPDATE_SITE_TITLE` message is one-off, I decided to style it separately, in order to not overcomplicate the logic in `stylizeQuickStartPrompt`.